packages/client/src/components/SearchBarPopup/searchBarPopup.scss

```diff
@@ -36,12 +36,13 @@
         }
     }
     & .results-container {
-        height: 100%;
+        height: 40vh;
```

Comment:
When `resultOpen` is true, this will expand the results section to this height. Needed in order to display `No results` text in the middle of the results section